### DISCUSSION TITLE
Fehlermeldung beim Angemeldet bleiben und veröffentlichen eines Beitrages

### DISCUSSION THREAD

**Discussion by Pflegecraft** [ophian/styx#15] [Q&A]
Hallo ich bekomme wenn ich den Hacken bei Eingelogt bleiben beim veröffentlichen eines Beitrages:
`Ihr Browser hat keinen gültigen HTTP-Referrer übermittelt. Dies kann entweder daher kommen, dass Ihr Browser/Proxy nicht korrekt konfiguriert ist, oder dass Sie Opfer einer "Cross Site Request Forgery (XSRF)" waren, mit der man Sie zu ungewollten Änderungen zwingen wollte. Die angeforderte Aktion konnte daher nicht durchgeführt werden`

Also so ganz ist der Fehler doch noch nicht weg.

Lieben Gruß
Flo

**Comment by ophian** [comment]
Auf welchen Fehler beziehst du dich?

Du sagst also du hast den Haken für das permanente Login gesetzt und bekommst 

1. dadurch
2. beim Veröffentlichen eines Eintrages

diese Fehlermeldung?

 Wenn du nicht so eingeloggt wärest, ginge es?
 Und wenn du danach das permente Login erneut setzt und einen weiteren Eintrag veröffentlichen willst, wieder nicht?

Um untergeschobene URL Veränderungen zu unterbinden werden verändernde links durch ein token abgesichert.
Mögliche Ursachen sind zum Beispiel:

1. Der Referer ist eine Seite, die auf einer alten Session beruht, zb. offengelassener tab aus alter Sitzung

2. Irgendetwas an der Serverkonfiguration oder Software hat sich geändert, so dass PHP-Sitzungen nicht mehr richtig funktionieren (falsch konfigurierter /tmp-Speicher, geänderte Serverkomponenten, neue php.ini, ...)

3. Die Browser oder Browserkonfiguration wurde geändert, oder Ad- oder Script-Blocking-Plugins oder Antivirus- oder sonstige Antispy-Software installiert, die jene Cookies nicht mehr akzeptiert, die Styx benötigt, um dich als eingeloggt zu erkennen und authorisiert zu betrachten (zb secured cookies) .

Siehe auch [hier im Buch](https://ophian.github.io/book/#U413)

**Comment by Pflegecraft** [comment]
Okay ja der Fehler lag in meinen Browser einstellungen ich habe das Plugin NoRef zu strikt eingestellt gehabt :D ups sorry.
Aber vlt hat ja jemand das selbe Problem und wir können Ihn so damit helfen.

Einfach das Plugin dafür deaktivieren dann sollte es wieder laufen.
